packages/cli/internal/pkg/cli/logs_core.go

```diff
@@ -39,7 +39,7 @@ Valid time units are "s", "m", and "h".`
 	tailFlagDescription = "Follow the log output."
 )
 
-var println = fmt.Println
+var printLn = fmt.Println
```

Comment:
var printLn = func (args ...intervace{}) {
  _, _ = fmt.Println(args...)
}

This change will make the rest of the code a bit tidier.